pyomo/contrib/mindtpy/util.py

```diff
@@ -641,3 +642,49 @@ def update_gap(solve_data):
     solve_data.abs_gap = solve_data.UB - solve_data.LB
     solve_data.rel_gap = (solve_data.UB - solve_data.LB)/(abs(
         solve_data.UB if solve_data.objective_sense == minimize else solve_data.LB) + 1E-10)
+
+
+def update_dual_bound(solve_data, bound_value):
+    '''
+    Call after solve relaxed problem, including relaxed NLP and MIP master problem.
+    Use the optimal primal bound of the relaxed problem to update the dual bound.
+    '''
```

Comment:
I thought we were using the documentation of functions according to Rahul's previous PR (saying what the args and returns are). This is a good moment to push for documentation following Google's style https://google.github.io/styleguide/pyguide.html (particularly section 3.8.3)